.gitignore

```diff
@@ -80,3 +80,12 @@ tests/test_cases/*/transcript
 examples/*/tests/modelsim.ini
 examples/*/tests/transcript
 *.wlf
+
+# Riviera
+tests/test_cases/*/library.cfg
+tests/test_cases/*/dataset.asdb
+tests/test_cases/*/compile
+examples/*/tests/library.cfg
+examples/*/tests/dataset.asdb
+examples/*/tests/compile
+examples/*/tests/*.mgf
```

Comment:
Any reason this one is here but not in the `clean` or `test_cases` paths?

Reply:
Legacy. Sorry.